adoc/SAP-EIC-General.adoc

```diff
@@ -17,6 +17,14 @@ Other versions of {metallb} or {cm} can be used, but they may not have been test
 # end::disclaimer-production-versions[]
 
 
+# tag::disclaimer-production-versions-harvester[]
```

Comment:
I'm not sure if we rather shall drop the last sentence and put it in the *-Main.adoc instead of adding a whole new tag with duplicated documentation.
Let's discuss!